File: archive/k/kotlin/evenOdd.kt
fun main(args: Array<String>) {
  var num = readLine()!!.toIntOrNull()
  if(num == null){
    print("Usage: please input a number")
    return
  }
  if(num%2 == 0)
  {
    println("Even")
  }
  else
  {
    println("Odd")
  }
}

File: archive/o/ocaml/even-odd.ml
let even_odd n = if n mod 2 = 0 then "Even" else "Odd"
let parse_args = function [| _; n |] -> int_of_string_opt n | _ -> None

let () =
  print_endline
    (match parse_args Sys.argv with
    | Some num -> even_odd num
    | None -> "Usage: please input a number")
